third_party/v8/src/compiler/bytecode-analysis.cc

```diff
@@ -225,7 +225,7 @@ void UpdateOutLiveness(Bytecode bytecode, BytecodeLivenessState* out_liveness,
     int target_offset = accessor.GetJumpTargetOffset();
     out_liveness->Union(*liveness_map.GetInLiveness(target_offset));
   } else if (Bytecodes::IsSwitch(bytecode)) {
-    for (const auto& entry : accessor.GetJumpTableTargetOffsets()) {
+    for (auto entry : accessor.GetJumpTableTargetOffsets()) {
```

Comment:
What's this for?

Reply:
base iterator typing was changed, so how we accessed it needed to change. I believe this is the same thing that's happening upstream now